.github/workflows/website.yml

```diff
@@ -0,0 +1,133 @@
+name: Documentation
+
+on:
+  # TODO: Consider running on schedule instead of on push
```

Comment:
I'll drop this TODO in the next push. We can discuss this in a separate issue rather than committing the idea into the repo.